src/sentry/conf/server.py

```diff
@@ -637,6 +637,10 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue("sleep", routing_key="sleep"),
     Queue("stats", routing_key="stats"),
     Queue("subscriptions", routing_key="subscriptions"),
+    Queue(
+        "symbolicator.compute_low_priority_queue",
```

Comment:
using `symbolicate_event` specifically might be strange because we do already have an `events` grouping, and an `events.symbolicate_event` already as well. 

i'm seeing two general patterns in the naming scheme for queues: 
- plural forms of nouns
- some verb for the action being taken
based on those two patterns, we have `symbolications.compute_low_priority_queue` or `symbolicate.compute_low_priority_queue`. thoughts?

Reply:
good pattern spotting.  i guess `symbolications.compute_low_priority_queue` is the most consistent one.  Though having `_queue` tagged on the end seems out of place.  Maybe `symbolications.compute_low_priority_selection`?